manifests/manifest-prod.yml

```diff
@@ -0,0 +1,11 @@
+applications:
+- name: respondent-home-ui
+  disk_quota: 1G
+  health-check-http-endpoint: /info
+  health-check-type: http
+  instances: 2
+  memory: 512M
```

Comment:
This will be upped from the current 1 instance, 256M. Do we need that increase now?

Reply:
If it's good enough for r-ops, it's good enough for the public 😉